### DISCUSSION TITLE
Why the exported mmseg onnx model output dtype is int64？

### DISCUSSION THREAD

**Discussion by ruisv** [open-mmlab/mmdeploy#1500] [Q&A] [answered]
Such as:
<img width="935" alt="image" src="https://user-images.githubusercontent.com/27017322/206333119-760b8160-003a-4eb2-bc51-5de6d22517dc.png">

Int64 model has very low precision in inference,
How do i export float32 model?
Thanks.

**Comment by lvhan028** [comment, marked as answer]
I think PR #1379 is what you want.
